=== Suite started at 2024-10-28 22:13:28
=== Starting test, 4 test cases
=cases         4
=user          otptest
=host          3ab6de50d313
=hosts         3ab6de50d313 
=emulator_vsn  15.1.2
=emulator      beam
=otp_release   28
=started       2024-10-28 22:13:28
=case          config_func_error_1_SUITE:init_per_suite
=logfile       config_func_error_1_suite.init_per_suite.html
=started       2024-10-28 22:13:28
=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:exit_in_iptc
=logfile       config_func_error_1_suite.exit_in_iptc.html
=started       2024-10-28 22:13:28
=== ERROR! config_func_error_1_SUITE:init_per_testcase(exit_in_iptc, [{watchdog,
                                                                   <0.3731.0>},
                                                                  {tc_logfile,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/config_func_error_1_suite.exit_in_iptc.html"},
                                                                  {tc_group_properties,
                                                                   []},
                                                                  {tc_group_path,
                                                                   []},
                                                                  {data_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                  {priv_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/log_private/"}]) failed!
	Reason: {'EXIT',{nocatch,generated_throw}}

=ended         2024-10-28 22:13:28
=result        auto_skipped: {failed,
                                 {config_func_error_1_SUITE,
                                     init_per_testcase,
                                     {'EXIT',{nocatch,generated_throw}}}}
=elapsed       0.000000s
=== *** SKIPPED test case 1 of 4 ***
=case          config_func_error_1_SUITE:exit_in_eptc
=logfile       config_func_error_1_suite.exit_in_eptc.html
=started       2024-10-28 22:13:28
=== WARNING! config_func_error_1_SUITE:end_per_testcase(exit_in_eptc, [{tc_status,
                                                                    ok},
                                                                   {watchdog,
                                                                    <0.3736.0>},
                                                                   {tc_logfile,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/config_func_error_1_suite.exit_in_eptc.html"},
                                                                   {tc_group_properties,
                                                                    []},
                                                                   {tc_group_path,
                                                                    []},
                                                                   {data_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                   {priv_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/log_private/"}]) failed!
	Reason: {'EXIT',{badmatch,undefined}}

=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:init_per_group
=logfile       config_func_error_1_suite.init_per_group.html
=group_props   [{name,g1}]
=started       2024-10-28 22:13:28
=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:exit_in_iptc
=logfile       config_func_error_1_suite.exit_in_iptc.1187.html
=started       2024-10-28 22:13:28
=== ERROR! config_func_error_1_SUITE:init_per_testcase(exit_in_iptc, [{watchdog,
                                                                   <0.3743.0>},
                                                                  {tc_logfile,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/config_func_error_1_suite.exit_in_iptc.1187.html"},
                                                                  {tc_group_properties,
                                                                   [{name,
                                                                     g1}]},
                                                                  {tc_group_path,
                                                                   []},
                                                                  {data_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                  {priv_dir,
                                                                   "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/log_private/"}]) failed!
	Reason: {'EXIT',{badmatch,undefined}}

=ended         2024-10-28 22:13:28
=result        auto_skipped: {failed,
                                 {config_func_error_1_SUITE,
                                     init_per_testcase,
                                     {'EXIT',{badmatch,undefined}}}}
=elapsed       0.000000s
=== *** SKIPPED test case 3 of 4 ***
=case          config_func_error_1_SUITE:end_per_group
=logfile       config_func_error_1_suite.end_per_group.html
=group_props   [{name,g1}]
=started       2024-10-28 22:13:28
=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=group_time    0.058s
=case          config_func_error_1_SUITE:init_per_group
=logfile       config_func_error_1_suite.init_per_group.7330.html
=group_props   [{name,g2}]
=started       2024-10-28 22:13:28
=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:exit_in_eptc
=logfile       config_func_error_1_suite.exit_in_eptc.7362.html
=started       2024-10-28 22:13:28
=== WARNING! config_func_error_1_SUITE:end_per_testcase(exit_in_eptc, [{tc_status,
                                                                    ok},
                                                                   {watchdog,
                                                                    <0.3752.0>},
                                                                   {tc_logfile,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/config_func_error_1_suite.exit_in_eptc.7362.html"},
                                                                   {tc_group_properties,
                                                                    [{name,
                                                                      g2}]},
                                                                   {tc_group_path,
                                                                    []},
                                                                   {data_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/config_func_error_1_SUITE_data/"},
                                                                   {priv_dir,
                                                                    "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@3ab6de50d313.2024-10-28_22.01.52/make_test_dir.common_test_test.logs/run.2024-10-28_22.01.55/log_private/ct_run.ct@3ab6de50d313.2024-10-28_22.13.28/ct_error_SUITE_data.error.config_func_error_1_SUITE.logs/run.2024-10-28_22.13.28/log_private/"}]) failed!
	Reason: {'EXIT',{case_clause,'CASE_CLAUSE'}}

=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=case          config_func_error_1_SUITE:end_per_group
=logfile       config_func_error_1_suite.end_per_group.7426.html
=group_props   [{name,g2}]
=started       2024-10-28 22:13:28
=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=group_time    0.058s
=case          config_func_error_1_SUITE:end_per_suite
=logfile       config_func_error_1_suite.end_per_suite.html
=started       2024-10-28 22:13:28
=ended         2024-10-28 22:13:28
=result        ok
=elapsed       0.000000s
=group_time    0.263s
=== TEST COMPLETE, 2 ok, 0 failed, 2 skipped of 4 test cases

=finished      2024-10-28 22:13:28
=failed        0
=successful    2
=user_skipped  0
=auto_skipped  2
=elapsed_time  319530
